README.md

```diff
@@ -280,6 +280,8 @@ tuned for maximum accuracy. For detailed results and more info about the benchma
 
 ### Imagenet
 
+**Note**: All results are obtained with SimCLR evaluation settings.
```

Comment:
Maybe a 1-2 sentence summary of what that means would be great + a reference to which paper it introduced?

Reply:
What igor said